.github/workflows/scheduled-class-upload.yml

```
@@ -32,7 +32,27 @@ jobs:
 
       - name: 'Upload classes from primary vimeo account'
         run: npm run tools:classes:last
-        
+
+      - name: Slack Notification
+        id: slack
+        if: ${{ failure() }}
+        uses: slackapi/slack-github-action@v1.19.0
+        with: 
+          channel-id: 'C01GKL43JRY' ##team-tech-updates
+          slack-message: ':alert: Upload classes ${{ job.status }}'
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_BOT_TOKEN }}
+
+      - name: Slack Notification
+        id: slack
+        if: ${{ success() }}
+        uses: slackapi/slack-github-action@v1.19.0
+        with: 
+          channel-id: 'C01GKL43JRY' ##team-tech-updates
+          slack-message: 'Successful upload classes :rocket:'
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_BOT_TOKEN }}
```

Viendo los ejemplos de la [doc](https://github.com/slackapi/slack-github-action), esto no debería ser mas bien así 🤔 
```suggestion
          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
```

---

Buena vista. Gracias. cambiado.